v0/pipeline/steps/cd/k8s-traffic-routing-rule.yaml

```diff
@@ -1,4 +1,4 @@
-title: TrafficRouteRule
+title: K8sTrafficRoutingRouteRule
```

Comment:
Shouldn't it be K8sTrafficRoutingRule instead of K8sTrafficRoutingRouteRule?

Reply:
No. Title should be K8sTrafficRoutingRouteRule. We should actually change the file name as well.